### ISSUE TITLE
Замечания к тестовому заданию в компанию Уно-Софт

### ISSUE THREAD

**Issue by ADsty** [vlsolodilov/SpeedTestApp#1]
Добрый день, во время проверки Вашего тестового задания обнаружилось несколько багов, которые следует поправить:
1. Тема приложения не меняется с темной на светлую, если системной выставлена темная;
2. Если в настройках выбрать расчет только одного показателя, то приложение все равно будет рассчитывать второй (в ТЗ указано что должны рассчитываться только выбранные);
3. Перед концом теста состояние меняется на "Ошибка теста" даже если тест прошел успешно;
4. Пункт "URL сервера Speed тест" не выполнен, следует указать его на первом экране, если его нельзя поменять;
5. При перевороте экрана UI приложения полностью ломается, тест становится невозможно запустить.

Пункты 1-2 и 4 обязательны к исправлению, поскольку указаны в ТЗ, в пунктах 3 и 5 можно обойтись простейшим способом решения, который придет на ум.

**Comment by vlsolodilov** [comment]
@ADsty Добрый день.
1, 4, 5 исправил
2, 3 - изменил api теста скорости, предыдущее не позволяло проводить только один тест

**Comment by ADsty** [comment]
@vlsolodilov Будет возможность сбилдить новый APK для проверки изменений?

**Comment by vlsolodilov** [comment]
@ADsty конечно, [вот](https://drive.google.com/file/d/1Vb3mc9e-Chxhaz0MgTDPEJFYctEgyz3X/view?usp=sharing)
